release/release_workflow.wdl

```diff
@@ -113,8 +106,10 @@ task versionPrep {
     }
 
     command <<<
+      set -e
+
       which jq || brew install jq
-      curl -s https://api.github.com/repos/~{organization}/cromwell/releases/latest | jq --raw-output '.tag_name' > version
+      curl --fail -v -s https://api.github.com/repos/~{organization}/cromwell/releases/latest | jq --raw-output '.tag_name' > version
```

Comment:
what's the net effect of `--fail` (fail silently) and `-v` (be verbose)?

Reply:
- `--fail` forces curl to change its exit code if something goes wrong
- `-v` means that we don't lose all context if that happens